ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -663,3 +645,34 @@ def setup_ceph_debug():
     )
     log.info("Setting Ceph to work in debug log level using a new ConfigMap resource")
     run_cmd(f"oc create -f {ceph_configmap_yaml.name}")
+
+
+def add_capacity_ui(osd_size_capacity_requested):
+    """
+    Add storage capacity to the cluster
+
+    Args:
+        osd_size_capacity_requested(int): Requested osd size capacity
+
+    """
+    ocp_version = get_ocp_version()
+    platform = config.ENV_DATA.get("platform", "").lower()
+    is_lso = config.DEPLOYMENT.get("local_storage")
+    if (
+        ocp_version == "4.7"
+        and (
+            platform == constants.AWS_PLATFORM or platform == constants.VSPHERE_PLATFORM
+        )
+        and (not is_lso)
+    ):
+        try:
+            logging.info("Add capacity via UI")
+            setup_ui = login_ui()
+            add_ui_obj = AddReplaceDeviceUI(setup_ui)
+            add_ui_obj.add_capacity_ui()
+            close_browser(setup_ui)
+        except Exception as e:
+            logging.error(e)
+            add_capacity(osd_size_capacity_requested)
```

Comment:
I think we can add here another log message, "Add capacity via UI failed, try to add capacity via cli" or something similar, before executing the line `add_capacity(osd_size_capacity_requested)`, so it will be more clear.